modules/cnf-about-ptp-events-using-ptp-event-producer-with-o-ran-api.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * networking/ptp/ptp-cloud-events-consumer-dev-reference-v2.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-about-ptp-events-using-ptp-event-producer-with-o-ran-api_{context}"]
+= Getting PTP events using the PTP events REST API v2
+
+Applications subscribe to PTP events by using an O-RAN v3 compatible REST API in the producer-side cloud event proxy sidecar.
+The `cloud-event-proxy` sidecar container can access the same resources as the primary application container without using any of the resources of the primary application and with no significant latency.
+
+.Overview of consuming PTP fast events from the PTP event producer REST API
+image::ptp-tbd-TELCODOCS-1862-O-RAN-API.png[Overview of consuming PTP fast events from the PTP event producer REST API]
```

Comment:
should we just remove the crossed out boxes. CC @aneeshkp

Reply:
Yes sorry we will. This is a draft image. Plan is to remove